### PR TITLE
Enable syslog collection for AKS - Azure Container Insights

### PR DESCRIPTION
**Pull request by MattLeach25**
## PR Summary

syslog collection has recently been announced as a preview feature in Azure Container Insights. This has been added as a feature.

#517

image

## PR Checklist

- [x] PR has a meaningful title
- [x] Summarized changes
- [x] This PR is ready to merge and is not **Work in Progress**
- [x] Link to a filed issue
- [x] Screenshot of UI changes (if PR includes UI changes)

### CODE REVIEW

File: bicep/main.bicep [hunk 1]
```diff
@@ -1495,6 +1495,8 @@ param AksDiagCategories array = [
   'guard'
 ]
 
+param enableSysLog bool = false
```
Comment (Gordonby) [inline bicep/main.bicep:1498]: Can we get metadata descripton?

File: bicep/main.bicep [hunk 2]
```diff
@@ -1513,6 +1515,102 @@ resource AksDiags 'Microsoft.Insights/diagnosticSettings@2021-05-01-preview' =
...
+        {
+          name: 'ciworkspace'
+          workspaceResourceId: aks_law.id
+        }
+      ]
+    }
+  }
+}
+
+// resource aks 'Microsoft.ContainerService/managedClusters@2022-11-02-preview' existing = {
```
Comment (Gordonby) [inline bicep/main.bicep:1601]: can you get rid of this comment

File: bicep/main.bicep [hunk 3]
```diff
@@ -1513,6 +1515,102 @@ resource AksDiags 'Microsoft.Insights/diagnosticSettings@2021-05-01-preview' =
...
+//   name: 'aks-az-k8s-f3pi'
+// }
+
+resource association 'Microsoft.Insights/dataCollectionRuleAssociations@2021-09-01-preview' = {
```
Comment (Gordonby) [inline bicep/main.bicep:1605]: what's test6?

**Review by Gordonby — CHANGES_REQUESTED** [review]

File: bicep/main.bicep (hunk 1 above)
Comment (MattLeach25) [reply bicep/main.bicep:1498]: done

**Review by Gordonby — APPROVED** [review]

**Pull request merged**